<launch>
    <arg name="visualization" default="true"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="terrain_planner" type="test_dubins_planner" name="dubins_planner" output="screen">
    </node>
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find terrain_planner)/launch/config_dubins.rviz" />
    </group>
</launch>
